### PR TITLE
Feat: REPLAT-4828 Update colours in In Depth article

### PR DESCRIPTION
**Pull request by jimhunty**
Support for the TPA values of `backgroundColour` and `textColour`.

`dropcapsDisabled`, `savingEnabled` and `sharingEnabled` are added to the queries, but not sure of the requirements for these. Didnt want them to be missed but happy to remove them if everyone agrees.

### CODE REVIEW

File: packages/article-in-depth/src/article-header/article-header.js [hunk 1]
```diff
@@ -19,31 +19,46 @@ const ArticleHeader = ({
...
-      <View
-        style={[
-          styles.container,
-          { backgroundColor: backgroundColour, width: "100%" }
-        ]}
-      >
-        <Label color={textColour} label={label} />
-        <Text
+}) => {
+  const bgColour =
```
Comment (nathggns) [inline packages/article-in-depth/src/article-header/article-header.js]: This code to convert an rgba colour to a string is duplicated four times, probably a good candidate for abstraction.

File: packages/provider/__tests__/__snapshots__/article-provider.test.js.snap [hunk 2]
```diff
@@ -188,12 +199,24 @@ Object {
       },
     ],
   },
+  "savingEnabled": true,
```
[the remark this answers is not in the document]
Comment (jimhunty) [reply packages/provider/__tests__/__snapshots__/article-provider.test.js.snap]: Yep - mentioned that in the description. I will remove.

File: packages/provider/__tests__/__snapshots__/article-provider.test.js.snap [hunk 3]
```diff
@@ -31,6 +41,7 @@ Object {
       "name": "paragraph",
     },
   ],
+  "dropcapsDisabled": false,
```
[the remark this answers is not in the document]
Comment (nehasri89) [reply packages/provider/__tests__/__snapshots__/article-provider.test.js.snap]: I can confirm this is the same name 👍

File: packages/article-in-depth/src/article-header/article-header.js [hunk 4]
```diff
@@ -19,31 +19,46 @@ const ArticleHeader = ({
...
+    backgroundColour && backgroundColour.rgba
+      ? `rgba(${backgroundColour.rgba.red}, ${backgroundColour.rgba.green}, ${
+          backgroundColour.rgba.blue
+        }, ${backgroundColour.rgba.alpha})`
+      : null;
+  const txtColour =
+    textColour && textColour.rgba
+      ? `rgba(${textColour.rgba.red}, ${textColour.rgba.green}, ${
```
[the remark this answers is not in the document]
Comment (jimhunty) [reply packages/article-in-depth/src/article-header/article-header.js]: Now converted to Hex and being passed through where needed

File: packages/article-in-depth/src/article-header/article-header.js [hunk 5]
```diff
@@ -20,31 +21,37 @@ const ArticleHeader = ({
...
-          styles.container,
-          { backgroundColor: backgroundColour, width: "100%" }
-        ]}
-      >
-        <Label color={textColour} isVideo={hasVideo} label={label} />
-        <Text
+}) => {
+  const bgColour = gqlRgbaToStyle(backgroundColour);
+  const txtColour = gqlRgbaToStyle(textColour);
+  const textHex = gqlRgbaToHex(textColour);
```
Comment (tuncaulubilge) [inline packages/article-in-depth/src/article-header/article-header.js]: Can you do this conversion inside the component itself, just before passing it into the svg? That way, this use case stays contained inside the component itself, so from an component api perspective, flags will be able accept colour in any format, reducing the chance of any errors in the future

Comment (tuncaulubilge) [reply packages/article-in-depth/src/article-header/article-header.js]: In other words, the whole app can use either rgba or hex anywhere, and the svg package sorts its own bug within the package itself. What do you think?

Comment (jimhunty) [reply packages/article-in-depth/src/article-header/article-header.js]: Yea that is a much better idea

**Pull request merged**